A38-admin-interface-api.md

```diff
@@ -1,57 +1,36 @@
 Admin Interface API
 ----
 * Author(s): lidiz
-* Approver: markroth
-* Status: Draft
+* Approver: markdroth
+* Status: In-Review
 * Implemented in: C++, Golang, Java
-* Last updated: 2021-01-27
-* Discussion at: TBD
+* Last updated: 2021-02-05
+* Discussion at: https://groups.google.com/g/grpc-io/c/0B2f9ha5HoM
 
 ## Abstract
 
-This proposal describes a convenience API in each gRPC language to improve the usability of creating a gRPC server with admin services.
+This proposal describes a convenience API in each gRPC language to improve the usability of creating a gRPC server with admin services to expose states in the gRPC library.
 
 
 ## Background
 
-gRPC Admin Interface is part of the Proxyless Service Mesh Observability project which includes new gRPC admin services and a CLI tool (yet to release). The admin services are responsible for exposing states of the gRPC library via gRPC protocol, and the CLI tool will help users to better interpret the fetched data from the gRPC application. Currently, the existing admin server should at least expose two admin services:
+### Services Exposing gRPC Internal States
```

Comment:
I don't think this subsection header is needed, since this is now the only subsection in the "Background" section.